.github/workflows/gcovr-ci-job.yml

```diff
@@ -87,28 +92,6 @@ jobs:
           # Set the CC environment
           echo "CC=${{ inputs.gcc }}" >> $GITHUB_ENV
 
-          if [ -z "$(sed -n '/Next Release/ p' CHANGELOG.rst)" ] ; then
```

Comment:
@Spacetown if this ever comes up, here's how I make `setuptools-scm` produce a version string without the local part in contexts where this is needed (merging to the default branch in my case): https://github.com/cherrypy/cheroot/blob/fdaa24d/.github/reusables/tox-dev/workflow/reusable-tox/hooks/prepare-for-tox-run/action.yml#L82-L125 + https://github.com/cherrypy/cheroot/blob/fdaa24d/.github/workflows/ci-cd.yml#L245-L265.